modules/infrastructure-moving-monitoring.adoc

```diff
@@ -5,10 +5,65 @@
 [id="infrastructure-moving-monitoring-{context}"]
 = Moving the monitoring solution
 
-[IMPORTANT]
-====
-The monitoring solution uses Cluster Version Operator (CVO) to create the
-ConfigMap that the monitoring Operator uses to determine how to deploy its
-resources. Because it uses the CVO and users cannot modify the CVO, you cannot
-change where the Operator deploys resources.
-====
\ No newline at end of file
+
+By default, a Prometheus, Grafana, and AlertManager stack is deployed to
+provide cluster monitoring. It is managed by the Cluster Monitoring Operator,
+and to move its components to different machines, you apply a custom ConfigMap.
+
+.Procedure
+
+. Optionally, review the default ConfigMap for the monitoring stack:
++
+----
+$ oc get configmap cluster-monitoring-config -n openshift-monitoring
```

Comment:
no need for this step, cluster-monitoring-config configmap is not exist when the cluster is UP, we could create the configmap by the following yaml file or edit the cluster-monitoring-config configmap if it is created